### DISCUSSION TITLE
Confusion over GUI terminology

### DISCUSSION THREAD

**Discussion by MaulMachine** [bcssov/IronyModManager#254] [Q&A]
Hello,

I started using your mod manager the other day to help me resolve a conflict in a mod pack for Stellaris (no luck, but I'm still trying) and I don't quite understand a thing I see in your  mod screen.  What does the Patch Mod: Enabled box mean?  Why does it appear in some collection mods and not others?

Thanks,

MaulMachine

**Comment by bcssov** [comment]
If you run conflict solver and resolve stuff it's generated. If you don't it's not generated so there's nothing to apply. As for the patch mods it's the stuff that contains all your conflict solver resolutions. I don't think I need to explain what enabled or disabled means though.

**Reply by MaulMachine** [reply]
Thank you.

If I reconcile all file differences and game keeps right on crashing, what else might be causing it?

**Reply by bcssov** [reply]
Mod incompatibility, error in merging and lots of other reasons.

**Reply by MaulMachine** [reply]
I see.  That's disappointing.

Thanks anyway.
